src/app/Plans/components/PlansTable.tsx

```diff
@@ -66,10 +77,33 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
 
   currentPageItems.forEach((plan: IPlan) => {
     let buttonText: string | React.ReactNode;
-    if (plan.status.conditions.every((condition) => condition.type === 'Ready')) {
+    let isStatusReady = false;
+    let title = '';
+    let variant: ProgressVariant | undefined;
+
+    if (
+      plan.status.conditions.every((condition) => condition.type === PlanStatusConditionsType.ready)
```

Comment:
Looking good, Gilles. One thing -  I am wondering how this logic reads- I think there can be a case where there is a ready condition & some additional warning conditions in the conditions array?

Reply:
Thanks for the review; 
I replaced the conditions check to search for only one valid case.